src/transform-urls.js

```diff
@@ -18,7 +18,14 @@ export const normalizeMapboxUrl = (url, accessToken) => {
 };
 
 export const isMaptilerUrl = url => {
-  let nextUrl = new URL(url);
+  if (!url) return false;
+  let nextUrl;
+  try {
+    nextUrl = new URL(url);
+  } catch (err) {
+    return false;
+  }
+
```

Comment:
This isn't directly related but this failed on various functionalities we had and didn't test for when the change went in (eg pulling a local style with a relative url)